lib/source_file.cpp

```diff
@@ -242,13 +265,29 @@ cstring SourceInfo::toSourceFragment() const {
     return InputSources::instance->getSourceFragment(*this);
 }
 
+cstring SourceInfo::toSourceFragment2() const {
+    return InputSources::instance->getSourceFragment2(*this);
+}
+
 cstring SourceInfo::toPositionString() const {
     if (!this->isValid())
         return "";
     SourceFileLine position = InputSources::instance->getSourceLine(this->start.getLineNumber());
     return position.toString();
 }
 
+cstring SourceInfo::toSourcePositionData(unsigned *outLineNumber,
+                                         unsigned *outColumnNumber) const {
+    SourceFileLine position = InputSources::instance->getSourceLine(this->start.getLineNumber());
```

Comment:
@mbudiu-vmw Is this a change you would strongly prefer be part of this PR, or is it OK for it to wait until a later full pass through this file?